python/ray/actor.py

```diff
@@ -418,7 +418,8 @@ def options(self,
                 lifetime=None,
                 placement_group=None,
                 placement_group_bundle_index=-1,
-                placement_group_capture_child_tasks=None):
+                placement_group_capture_child_tasks=None,
+                override_worker_env=None):
```

Comment:
Let's call this `override_env` instead (`worker` is not a concept in the Ray API)

Reply:
Actually `override_environment_variables` is even clearer. Clarity is probably more important than verbosity here since this will not be used super often. Also we are already using `environment_variables` in the autoscaler.